Formula/adios2.rb

```diff
@@ -0,0 +1,81 @@
+class Adios2 < Formula
+  desc "Next generation of ADIOS developed in the Exascale Computing Program"
+  homepage "https://adios2.readthedocs.io"
+  url "https://github.com/ornladios/ADIOS2/archive/v2.5.0.tar.gz"
+  sha256 "7c8ff3bf5441dd662806df9650c56a669359cb0185ea232ecb3578de7b065329"
+  head "https://github.com/ornladios/ADIOS2.git", :branch => "master"
+
+  depends_on "cmake" => :build
+  depends_on "gcc" => :build
+  depends_on "c-blosc"
+  depends_on "libfabric"
+  depends_on "libpng"
+  depends_on "numpy"
+  depends_on "open-mpi"
+  depends_on "python"
+  depends_on "zeromq"
+  uses_from_macos "bzip2"
+
+  resource "mpi4py" do
+    url "https://files.pythonhosted.org/packages/04/f5/a615603ce4ab7f40b65dba63759455e3da610d9a155d4d4cece1d8fd6706/mpi4py-3.0.2.tar.gz"
+    sha256 "f8d629d1e3e3b7b89cb99d0e3bc5505e76cc42089829807950d5c56606ed48e0"
+  end
+
+  def install
+    resource("mpi4py").stage do
+      system "#{Formula["python"].opt_bin}/python3",
+             *Language::Python.setup_install_args(libexec)
+    end
+    xy = Language::Python.major_minor_version "#{Formula["python"].opt_bin}/python3"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python#{xy}/site-packages"
```

Comment:
This looks like it should work. I'm not sure if you need to pass anything to CMake, it should query the python path, no?